plugins/module_utils/waiters.py

```diff
@@ -250,6 +263,36 @@
                 },
             ]
         },
+        "VpcAvailable": {
```

Comment:
The same for doing this. https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Waiter.VpcAvailable
Does it makes sense to you?